config/base.toml

```diff
@@ -9,18 +9,23 @@ boundary_geography = "OA"
 nts_years = [2019, 2021, 2022]
 # NTS regions to use
 nts_regions = [
-    'Yorkshire and the Humber',
-    'North West',
-    'North East',
-    'East Midlands',
-    'West Midlands',
-    'East of England',
-    'South East',
-    'South West',
+    "Northern, Metropolitan",
+    # "Northern, Non-metropolitan",
+    "Yorkshire / Humberside, Metropolitan",
+    # "Yorkshire / Humberside, Non-metropolitan",
+    "East Midlands",
+    "East Anglia",
+    "South East (excluding London Boroughs)",
+    # "London Boroughs",
+    "South West",
+    "West Midlands, Metropolitan",
+    # "West Midlands, Non-metropolitan",
+    "North West, Metropolitan",
+    # "North West, Non-metropolitan",
 ]
 # nts day of the week to use
 # 1: Monday, 2: Tuesday, 3: Wednesday, 4: Thursday, 5: Friday, 6: Saturday, 7: Sunday
-nts_day_of_week = 3
+nts_days_of_week = [3]
 # what crs do we want the output to be in? (just add the number, e.g. 3857)
 output_crs = 3857
 
```

Comment:
Add the new parameters to the `base.toml` as being used to document config.

Reply:
Add `part_time_work_prob` and a description of what the parameter does and also the effect when `use_percentages = false/true`

Reply:
Opened #99

Reply:
Completed in [a6311af](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/90/commits/a6311af83c1fcf4df986e508dd69e3c0b5881dbc)